lms/djangoapps/ccx/plugins.py

```diff
@@ -31,6 +34,18 @@ def is_enabled(cls, course, user=None):
             # If ccx is not enable do not show ccx coach tab.
             return False
 
+        if (
```

Comment:
Please, add a comprehensive comment to understand why this is here. @anfbermudezme